recipes/renode-cores/helpers/renode_build_with_cmake.ps1

```diff
@@ -0,0 +1,68 @@
+$ErrorActionPreference = "Stop"
+
+$cpuCount = (Get-CimInstance Win32_Processor).NumberOfLogicalProcessors
+
+$env:PATH = "${env:BUILD_PREFIX}/Library/mingw-w64/bin;${env:BUILD_PREFIX}/Library/bin;${env:PREFIX}/Library/bin;${env:PREFIX}/bin;${env:PATH}"
+$env:SRC_DIR = $env:SRC_DIR
+
+$CMAKE = (Get-Command cmake).Source
+
+$CONFIGURATION = "Release"
+$HOST_ARCH = "i386"
+$CMAKE_COMMON = ""
```

Comment:
See my previous comments about not relying so much on variables and simplifying logic - I suspect as with the other helper scripts we probably don't actually need separate scripts if we simplify this enough.